### PR TITLE
Make COSMIC and Cohort sort descending first

### PR DESCRIPTION
**Pull request by adamabeshouse**
# Checks
- [x] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [x] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).
- [x] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)

### CODE REVIEW

File: src/shared/components/msktable/MSKTable.tsx [hunk 1]
```diff
@@ -20,13 +21,14 @@ export type Column<T> = {
     render:(data:T)=>JSX.Element;
     download?:(data:T)=>string;
     tooltip?:JSX.Element;
+    firstSortDirection?:SortDirection;
```
Comment (onursumer) [inline src/shared/components/msktable/MSKTable.tsx]: Why don't we name this one also `initialSortDirection`?

Comment (adamabeshouse) [inline src/shared/components/msktable/MSKTable.tsx]: feels a little confusing, initial feels like it implies it'll be like that on start-up

Comment (alisman) [inline src/shared/components/msktable/MSKTable.tsx]: What about defaultSortDirection?  I agree firstSortDirection is kind of strange.

Comment (onursumer) [inline src/shared/components/msktable/MSKTable.tsx]: default sounds better than initial!

Comment (adamabeshouse) [inline src/shared/components/msktable/MSKTable.tsx]: default it is!

**Comment by adamabeshouse** [comment]
[sort cosmic and cohort columns descending at first](https://trello.com/c/a29zM76b/214-sort-cosmic-and-cohort-columns-descending-at-first)

**Pull request merged**